pipeline_products_session/jwst-data-products-part3.ipynb

```diff
@@ -0,0 +1,901 @@
+    "* [countrate data](https://jwst-pipeline.readthedocs.io/en/stable/jwst/data_products/science_products.html#countrate-data-rate-and-rateints) produced by applying the Stage 1 (detector-level) corrections in order to compute count rates from the original accumulating signal ramps, identified by the suffix ```rate``` or ```rateints```\n",
+    "* [calibrated single exposures](https://jwst-pipeline.readthedocs.io/en/stable/jwst/data_products/science_products.html#calibrated-data-cal-and-calints), identified by the suffix ```cal```\n",
+    "* [resampled and/or combined exposures](https://jwst-pipeline.readthedocs.io/en/stable/jwst/data_products/science_products.html#resampled-2-d-data-i2d-and-s2d), identified by the suffixes ```i2d``` or ```s2d```\n",
+    "* [extracted spectroscopic 1D data](https://jwst-pipeline.readthedocs.io/en/stable/jwst/data_products/science_products.html#extracted-1-d-spectroscopic-data-x1d-and-x1dints), identified by the suffixes ```x1d``` or ```c1d```\n",
+    "\n",
+    "In addition, there are also [several other products depending on the observing mode](https://jwst-pipeline.readthedocs.io/en/stable/jwst/data_products/science_products.html#source-catalog-cat), such as source and photometry catalogs, stacked PSF data, and NIRISS AMI derived data.  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Top of Page](#title_ID)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "3.<font color='white'>-</font>Example data for this exercise <a class=\"anchor\" id=\"example\"></a>\n",
+    "------------------\n",
+    "\n",
+    "For this module, we will use calibrated NIRCam simulated imaging and wide field slitless spectroscopy (WFSS) exposures that are stored in Box. Let's grab the data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "main_box_url =\"https://stsci.box.com/shared/static/5xsvwzwncpulnxobjxoodjihzeu5gyhf.fits\"\n",
+    "\n",
+    "combined_i2d_file = \"example_nircam_imaging_i2d\"\n",
+    "final_c1d_file = \"example_nircam_wfss_c1d.fits\"\n",
+    "output_catalog = \"example_nircam_imaging_cat.ecsv\"\n",
+    "\n",
+    "all_files = [combined_i2d_file, output_catalog,\n",
+    "             final_c1d_file]\n",
+    "\n",
+    "for file in all_files:\n",
+    "    demo_file = download_file(main_box_url1+file)\n",
+    "    \n",
+    "    # Save the file so that we can use it later\n",
+    "    with fits.open(demo_file) as f:\n",
+    "        f.writeto(file, overwrite=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Top of Page](#title_ID)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "4.<font color='white'>-</font>Data products: stage 3 (combined, rectified exposures)<a class=\"anchor\" id=\"stage3\"></a>\n",
```

Comment:
It might be helpful to define or describe "rectified" and "combined" here. e.g. "combined" = mosaicked.

Reply:
Fixed